src/seb/registered_tasks/mteb_retrieval.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any
+
+import datasets
+from mteb.abstasks import AbsTaskRetrieval
+
+
+class DanFever(AbsTaskRetrieval):
+    @property
+    def description(self) -> dict[str, Any]:
+        return {
+            "name": "DanFEVER",
+            "hf_hub_name": "strombergnlp/danfever",
+            "description": "A Danish dataset intended for misinformation research. It follows the same format as the English FEVER dataset.",
+            "reference": "https://aclanthology.org/2021.nodalida-main.47/",
+            "type": "Retrieval",
+            "category": "p2p",
+            "eval_splits": ["train"],
+            "eval_langs": ["da"],
+            "main_score": "ndcg_at_10",
+            "revision": "5d01e3f6a661d48e127ab5d7e3aaa0dc8331438a",
+        }
+
+    def load_data(self, **kwargs: dict):  # noqa: ARG002
+        """
+        Load dataset from HuggingFace hub
+        """
+        if self.data_loaded:
+            return
+
+        self.dataset: datasets.DatasetDict = datasets.load_dataset(
+            self.description["hf_hub_name"],
+            revision=self.description.get("revision"),
+        )  # type: ignore
+
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self) -> None:
+        """
+        and transform to a retrieval datset, which have the following attributes
+
+        self.corpus = Dict[doc_id, Dict[str, str]] #id => dict with document data like title and text
+        self.queries = Dict[query_id, str] #id => query
+        self.relevant_docs = Dict[query_id, Dict[[doc_id, score]]
+        """
+        self.corpus = {}
+        self.relevant_docs = {}
+        self.queries = {}
+        text2id = {}
+
+        for split in self.dataset:
+            self.corpus[split] = {}
+            self.relevant_docs[split] = {}
+            self.queries[split] = {}
+
+            ds = self.dataset[split]
+            claims = ds["claim"]
+            evidences = ds["evidence_extract"]
+            labels = ds["label"]
+            class_labels = ds.features["label"].names
+
+            for claim, evidence, label_id in zip(claims, evidences, labels):
+                claim_is_supported = class_labels[label_id] == "Supported"
+
+                sim = 1 if claim_is_supported else 0  # negative for refutes claims - is that what we want?
```

Comment:
@Muennighoff - The DanFEVER dataset is similar to the FEVER dataset. Just wanted to make sure that this dataset is constructed fairly similar to FEVER.

I use the claim as the query to all the evidence segments as the corpus. The relevance score is then determined by whether the claim is supported.

However, I am unsure if assigning 0 to "not supported" and "not enough evidence" is meaningful.

What are your thoughts?

Reply:
> However, I am unsure if assigning 0 to "not supported" and "not enough evidence" is meaningful.

If that's the same way it is done for FEVER, then I think it's okay!

Reply:
I am unsure how it is done for FEVER (can I find it somewhere?)

Reply:
This is what it says in BEIR, so it does seem like they everything that's not the evidence is a 0
`FEVER [60] The Fact Extraction and VERification dataset is collected to facilitate the automatic fact checking. We utilize the original paper splits as queries Q and retrieve evidences from the pre-processed Wikipedia Abstracts (June 2017 dump) as our corpus T`